matchms/exporting/save_as_mgf.py

```diff
@@ -8,6 +8,23 @@ def save_as_mgf(spectrums: List[Spectrum], filename: str):
 
     :py:attr:`~matchms.Spectrum.losses` of spectrum will not be saved.
 
+    Example:
+
+    .. code-block:: python
+
+        from matchms import Spectrum
```

Comment:
Also missing numpy here
```suggestion
        import numpy
        from matchms import Spectrum
```